docs-src/python/how-to-send-a-signal-from-a-client-in-python.md

```diff
@@ -9,8 +9,15 @@ tags:
   - python
 ---
 
-To send a Signal to a Workflow Execution from Client code, use the [`signal()`](https://python.temporal.io/temporalio.workflow.html#signal) method on the Workflow handle.
+To send a Signal from the Client, use the [`signal()`](https://python.temporal.io/temporalio.client.WorkflowHandle.html#signal) function on the Workflow handle.
+
+Use the [`get_workflow_handle()`](https://python.temporal.io/temporalio.client.Client.html#get_workflow_handle) method or the [`get_workflow_handle_for()`](https://python.temporal.io/temporalio.client.Client.html#get_workflow_handle_for) method to get a type safe Workflow handle by its Workflow Id.
```

Comment:
Note, this is the same handle returned from `start_workflow`, they don't have to get another handle if they already have it